src/main/java/core/basesyntax/geometry/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.geometry.figures;
+
+import core.basesyntax.geometry.Figure;
+
+public class Circle extends Figure {
+    private static final double NUMBER_P = 3.14;
+    private static final String NAME = "circle";
```

Comment:
it's a redundant constant, remove it and use Circle.class.getSimpleName() in draw method, the same for other implementations of FIgure